lib/Ed25519Signature2020.js

```diff
@@ -69,10 +71,10 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
     }
 
     const contexts = [].concat(document['@context'] || []);
-    if(!contexts.includes(SUITE_CONTEXT)) {
+    if(!contexts.includes(SUITE_CONTEXT_URL)) {
       throw new TypeError(
-        // eslint-disable-next-line max-len
-        `The document to be signed must contain this suite's @context, "${SUITE_CONTEXT}".`
+        `The document to be signed must contain this suite's @context, ` +
+          `"${SUITE_CONTEXT_URL}".`
```

Comment:
```suggestion
        'The document to be signed must contain this suite's @context, ' +
        `"${SUITE_CONTEXT_URL}".`
```

Reply:
We're we going to make it so we add the context to the document if it isn't present by default? Or is that not part of this PR?

Reply:
Yes. We're going to do the enforcing [here](https://github.com/digitalbazaar/jsonld-signatures/pull/139/files#diff-89df1fe4964bd8df8b9ebd53461b10c9c130e540c96f0ba6345be529d70ef269R59), in `jsigs.sign()`.